test/fakedata_generation.py

```diff
@@ -168,3 +169,75 @@ def _make_devkit_archive(root):
         _make_devkit_archive(root)
 
         yield root
+
+
+@contextlib.contextmanager
+def cityscapes_root():
+
+    def _make_image(file):
+        PIL.Image.fromarray(np.zeros((1024, 2048, 3), dtype=np.uint8)).save(file)
```

Comment:
I don't know how big the impact is but maybe we could use a smaller image size here instead of the original size to make the tests run faster?

Reply:
We are not performing any significant operations on the inputs, so this should not be a problem speed-wise.
But good point!